doc-site/docs/getting-started/installation.md

```diff
@@ -54,9 +54,14 @@ helm install cert-manager --namespace cert-manager --version v1.16.1 jetstack/ce
 Install the Paladin operator chart:
 
 ```bash
-helm upgrade --install paladin paladin/paladin-operator -n paladin --create-namespace \
-    --set paladin.namespace=paladin \
-    --set cert-manager.enabled=false
+helm upgrade --install paladin paladin/paladin-operator -n paladin --create-namespace
+```
+
+### Step 4: Deploy the CRs
+```
+curl -s https://api.github.com/repos/LF-Decentralized-Trust-labs/paladin/operator/config/samples   | \
+jq -r '.[] | select(.name | endswith(".yaml")) | .download_url' | \
+xargs -n 1 kubectl apply -n paladin -f
```

Comment:
Darn bit of a shame `kubectl` doesn't support folders...

But speaking from I've seen projects like Prometheus and cert-manager do - I wonder if this is an indicator to us that a "set" of samples should all one big YAML file ?

Then we could do `kubectl apply -f https://raw.githubusercontent.com/LF-Decentralized-Trust-labs/paladin/refs/heads/main/operator/config/samples/gettingstarted.yaml` instead which reads a bit nicer.

And it forces us to curate different example YAMLs, and make it clear what each is good for.

A giant `samples` folder of CRs which actually have dependencies on one another is a bit terse for an end-user / new contributer.

Reply:
I agree, let's first decide how we want to publish the CRs, once we have a decision I can consolidate the files into a single file.